setup.cfg

```diff
@@ -33,12 +33,14 @@ test =
     pytest-rerunfailures
     skypy-data @ https://github.com/skypyproject/skypy-data/archive/master.tar.gz
     specutils
+    speclite @ git+https://github.com/desihub/speclite.git
```

Comment:
This could probably be `speclite != 0.9` so that it will only skip the broken release.

Reply:
I have one problem with `tox` locally which might come from this part in the config file. If I run `tox` it apparently fails or just do not install `speclite` but it works here apparently?